src/services/json-api-datastore.service.ts

```diff
@@ -225,22 +226,17 @@ export class JsonApiDatastore {
         return jsonApiMeta;
     }
 
-    private getOptions(customHeaders?: Headers): RequestOptions {
-        let requestHeaders = new Headers();
-        requestHeaders.set('Accept', 'application/vnd.api+json');
-        requestHeaders.set('Content-Type', 'application/vnd.api+json');
-        if (this._headers) {
```

Comment:
Removing this means the user can not set "global" headers anymore. Was this intended?